src/SFA.DAS.EmployerIncentives.Api/HealthChecks/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Linq;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.Extensions.DependencyInjection;
+using SFA.DAS.EmployerIncentives.Api.Extensions;
+
+namespace SFA.DAS.EmployerIncentives.Api.HealthChecks
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static IServiceCollection AddDasHealthChecks(this IServiceCollection services)
+        {
+            services.AddHealthChecks()
+                .AddCheck<InnerApiHealthCheck>("Inner Employer Incentives API Health Check")
+                .AddCheck<CommitmentsV2ApiHealthCheck>("Commitments V2 API Health Check");
+
+            return services;
+        }
+
+        public static IApplicationBuilder UseDasHealthChecks(this IApplicationBuilder app)
+        {
+            app.UseHealthChecks("/ping"); 
+
+            return app.UseHealthChecks("/health", new HealthCheckOptions
```

Comment:
There already is a HealthCheckResponseWriter that should be used

Reply:
Can we discuss?